app/form/v1_0/fields/drug-use/drug-use-history.ts

```diff
@@ -30,9 +33,29 @@ const drugsReasonsForUse: FormWizard.Field = {
   labelClasses: utils.getMediumLabelClassFor(FieldType.CheckBox),
 }
 
-const drugsReasonsForUseDetails = FieldsFactory.detailsField({
-  parentField: drugsReasonsForUse,
-})
+const drugsReasonsForUseDetails: FormWizard.Field = {
+  text: 'Give details (optional)',
+  code: fieldCodeWith(drugsReasonsForUse.code, 'details'),
+  type: FieldType.TextArea,
+  validate: [
+    {
+      type: 'validateMaxLength',
+      fn: utils.validateMaxLength,
+      arguments: [characterLimits.default],
+      message: `Details must be ${characterLimits.default} characters or less`,
+    },
+  ],
+  transform(state): FormWizard.Field {
+    const usedInTheLastSixMonths = addDrugs.drugLastUsedFields.some(field => state.answers[field.code] === 'LAST_SIX')
+    const subject = state.session.subjectDetails as HandoverSubject
+    return {
+      ...this,
+      summary: {
+        text: `Details on why ${subject.givenName} ${usedInTheLastSixMonths ? 'uses' : 'used'} drugs`,
```

Comment:
I don't see where this has come from in the ACs for ARN-2702 or the Figma. Can you point it out please?